### ISSUE TITLE
primitive-math

### ISSUE THREAD

**Issue by KingMob** [clj-commons/meta#67]
@ztellman @slipset @piotr-yuxuan

Piotr requested some ns changes to an old, archived library of Zach's, `primitive-math`. They wanted to deprecate the top-level namespace and move code into a multi-segmented ns, as top-level namespaces cause some issues with using GraalVM in [clj-easy's graal-build-time](https://github.com/clj-easy/graal-build-time).

If we're going to make a change, I'd like to bring `primitive-math` under the clj-commons umbrella, if that's ok.

**Comment by danielcompton** [comment]
If Zach is happy to migrate primitive math, we should also take a look at removing this line:

https://github.com/ztellman/primitive-math/blob/7654d4c86a6a93639dd4d6d074a1fcb5ee64ebc2/project.clj#L15

I was doing some work on Java build stuff (can't remember if it was with Graal or `jdeps`) and having an old class file version caused issues. Sorry I can't remember the specifics, it wasn't a blocker for anything, but I think it might have prevented some optimisations.

**Comment by ztellman** [comment]
Yeah, absolutely happy to migrate it over.  

On a related note, it's been suggested that this repo I created when I worked at Factual could be transferred over, as well: https://github.com/Factual/durable-queue/issues/23.  I would expect exceedingly few changes to this repo over time, currently the only outstanding work are some new type-hints to better work with Java 13, but since I can't currently deploy new releases under the Factual org, it needs to be forked/transferred somewhere to make those small changes.
